.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Description
+
+Describe your changes here, and link to related issues if available.
+
+Reminder to run `python validate_po.py --extract` if you have added new translatable strings.
```

Comment:
Even if the person who sends the PR does not run it, the CI will run it, isn't it?

Reply:
CI will complain if they didn't run it, yes. But they have to run it/add it to their change themselves.